composer.json

```diff
@@ -23,7 +23,7 @@
   },
   "minimum-stability": "dev",
   "autoload-dev": {
-    "psr-0": {
+    "psr-4": {
```

Comment:
that is not how psr-4 works. please check other modules on the proper changes for it.